lms/envs/common.py

```diff
@@ -456,6 +456,10 @@
 OAUTH_EXPIRE_CONFIDENTIAL_CLIENT_DAYS = 365
 OAUTH_EXPIRE_PUBLIC_CLIENT_DAYS = 30
 
+################################## DJANGO OAUTH TOOLKIT #######################################
+
+OAUTH2_VALIDATOR_CLASS = 'lms.djangoapps.oauth_dispatch.dot_overrides.EdxOAuth2Validator'
```

Comment:
It is installed in CMS, but not actually used.  It is mostly just there to make imports happy.